src/registrar/assets/js/dja-collapse.js

```diff
@@ -38,8 +37,40 @@
                 fieldset.classList.add('collapsed');
             }
         };
-        document.querySelectorAll('.collapse-toggle--dotgov').forEach(function(el) {
+        document.querySelectorAll('.collapse-toggle--dgfieldset').forEach(function(el) {
             el.addEventListener('click', toggleFuncDotgov);
         });
     });
 }
+
+'use strict';
```

Comment:
Makes JS more strongly typed. I don't typically use it but the DJA folks did on their code, which I copied and edited for the fieldsets show/hide above. 

Again, this could all use standardizing.